### ISSUE TITLE
Enforce newline between linked control signatures? try/catch if/else 

### ISSUE THREAD

**Issue by JParkinson1991** [squizlabs/PHP_CodeSniffer#2948]
Hello,

Im wondering if there is a way to enforce that there is a newline between linked controls structures.

For example, enforcing the following code style on if, elseif, if

```
if (true) {
    echo "true"    
}
elseif (false) {
    echo "false:
}
else {
    echo "😕"
} 
```

And for try catch

```
try {
}
catch (Exception $e) {
}
```

I know when using PSR-12 standard the following is flagged as errors due to the sniff `Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace` and this sniff can be disabled. But disabling this sniff does not force the new line.

Anyone any ideas?

Thanks

**Comment by jrfnl** [comment]
Such a sniff does not exit in PHPCS itself. There is probably an external standard out there which will have a sniff for it, but I don't know of one off-hand.

Also see: #526 and #2025

**Comment by JParkinson1991** [comment]
Quick update from me, the constrol structure style i was after wasn't acutually the allman style. I did eventaully find a sniff for the style in my original issues. It was available as part of: https://www.drupal.org/project/coder

Lifting from i found i created a snif that address the issues: https://github.com/JParkinson1991/phpcodesniffer-standards/blob/master/src/Standards/JPSR12/Sniffs/ControlSignatures/MultiKeywordSniff.php

The sniff was then packaged into an adapated PSR-12 standard: https://github.com/JParkinson1991/phpcodesniffer-standards.
